index.html

```diff
@@ -14,41 +15,47 @@
     <header>
         <div class="prof-pic">
         </div>
-        <div id="media-links">
-          <div id="linked" class="media-link">
-            <a href="https://www.linkedin.com/in/scott-brenden-2a1b2297/" class="icon-linkedin2"></a>
-          </div>
-          <div id="facebook" class="media-link">
-            <a href="https://www.facebook.com/scott.brenden.1" class="icon-facebook"></a>
-          </div>
-          <div id="git" class="media-link">
-            <a href="https://github.com/ScottBrenden" class="icon-github"></a>
-          </div>
-          <div id="email" class="media-link">
-            <a href="#" class="icon-mail4"></a>
+        <div id="prof-info">
```

Comment:
Sweet job making your attributes "kabob-cased" instead of camel case as that's convention for HTML. 👍